src/main/java/org/commonwl/view/cwl/CWLService.java

```diff
@@ -448,9 +452,9 @@ public Workflow parseWorkflowWithCwltool(Workflow basicModel, Path workflowFile,
     }
     // Try to determine license
     ResultSet licenseResult = rdfService.getLicense(url);
-    String licenseLink = null;
+    String licenseLink;
     if (licenseResult.hasNext()) {
-      licenseLink = licenseResult.next().get("license").toString();
+      licenseLink = normaliseLicenseLink(licenseResult.next().get("license").toString());
```

Comment:
I **think** this is not yet covered? https://coveralls.io/builds/55567022/source?filename=src%2Fmain%2Fjava%2Forg%2Fcommonwl%2Fview%2Fcwl%2FCWLService.java#L457

Reply:
Did it